### PR TITLE
Add link about finding the configuration

### PR DESCRIPTION
**Pull request by ludeeus**
## Proposed change

Add a note about finding the configuration at the end of `{% configuration %}` blocks, this link points to "/docs/configuration/" which currently raised more questions than it answered, so this PR also adjust that page.

image

CC @DubhAd 

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [x] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: 
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: 

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [ ] The documentation follows the Home Assistant documentation [standards][].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

File: plugins/configuration.rb [hunk 1]
```diff
@@ -175,6 +175,7 @@ def render(context)
             platform: platform,
             converter: converter
           )}
+          <p class="configuration-link" title="Click here to find your configuration file"><a href="/docs/configuration/"><i class="icon-question" />Find your configuration</a>
```
Comment (ludeeus) [inline plugins/configuration.rb]:
```suggestion
          <p class="configuration-link" title="Click here to find your configuration file"><a href="/docs/configuration/" target="_blank"><i class="icon-question" />Find your configuration</a>
```

Currently, it takes over the current tab, maybe it's useful to have it open in a new tab to not disturb the user's "flow"?

Comment (frenck) [reply plugins/configuration.rb]: I think that is wise in this case (the new tab) 👍 

Unfortunately, this configuration block is "misused" in some places at this point still. But that should not stop this PR. Hopefully they will bubble up :)

File: source/_docs/configuration.markdown [hunk 2]
```diff
@@ -3,53 +3,40 @@ title: "Configuring Home Assistant"
 description: "Configuring Home Assistant."
 ---
 
-When launched for the first time, Home Assistant will create a default configuration file enabling the web interface and device discovery. It can take up to a minute after startup for your devices to be discovered and appear in the user interface.
+While you can configure most of Home Assistant directly from the user interface under {% my configuration %}, some parts needs you to access the configuration folder.
```
Comment (frenck) [inline source/_docs/configuration.markdown]:
```suggestion
While you can configure most of Home Assistant directly from the user interface under {% my configuration %}, some parts need you to access the configuration folder.
```

File: source/_docs/configuration.markdown [hunk 3]
```diff
@@ -3,53 +3,40 @@ title: "Configuring Home Assistant"
...
+## Reloading changes
 
-</div>
+You will have to restart Home Assistant for most changes to `configuration.yaml` to take effect.
+You can load changes to the following components without restarting, by using the UI. Navigate to {% my server_controls title="Configuration -> Server Control" %} and scrolling down to the YAML configuration reloading: [automations](/docs/automation/), [core (customize)](/docs/configuration/customizing-devices/), [groups](/integrations/group/), [history stats](/integrations/history_stats/), [HomeKit](/integrations/homekit/), [input_booleans](/integrations/input_boolean/), [input_datetimes](/integrations/input_datetime/), [input_numbers](/integrations/input_number/), [input_selects](/integrations/input_select/), [input_texts](/integrations/input_text/), [MQTT](/integrations/mqtt/), [persons](/integrations/person/), [scenes](/integrations/scene/), [scripts](/integrations/script/), [statistics](/integrations/statistics/), [template sensors](/integrations/template/), [timers](/integrations/timer/), [zones](/integrations/zone/), and more without restarting.
```
Comment (frenck) [inline source/_docs/configuration.markdown]:
```suggestion
You will have to restart Home Assistant for most changes to `configuration.yaml` to take effect.
You can load changes to the following integrations without restarting, by using the UI. Navigate to {% my server_controls title="Configuration -> Server Control" %} and scrolling down to the YAML configuration reloading: [automations](/docs/automation/), [core (customize)](/docs/configuration/customizing-devices/), [groups](/integrations/group/), [history stats](/integrations/history_stats/), [HomeKit](/integrations/homekit/), [input_booleans](/integrations/input_boolean/), [input_datetimes](/integrations/input_datetime/), [input_numbers](/integrations/input_number/), [input_selects](/integrations/input_select/), [input_texts](/integrations/input_text/), [MQTT](/integrations/mqtt/), [persons](/integrations/person/), [scenes](/integrations/scene/), [scripts](/integrations/script/), [statistics](/integrations/statistics/), [template sensors](/integrations/template/), [timers](/integrations/timer/), [zones](/integrations/zone/), and more without restarting.
```

File: plugins/configuration.rb (hunk 1 above)
Comment (ludeeus) [reply plugins/configuration.rb]: For those places a link might make sense anyway, if not I'm sure we will get feedback on it 👍

File: sass/custom/_paulus.scss [hunk 4]
```diff
@@ -626,6 +626,19 @@ div.config-vars {
...
+  .configuration-link {
+    font-size: 13px;
+
+    a {
+      color: #8792a2;
+      text-decoration: none;
+      border-bottom: #8792a2 dotted 0.5px;
+    }
+    i {
+      margin-right: 4px;
```
Comment (balloob) [inline sass/custom/_paulus.scss]: I think that you could just add a space between the icon and the text instead

File: plugins/configuration.rb [hunk 5]
```diff
@@ -175,6 +175,7 @@ def render(context)
             platform: platform,
             converter: converter
           )}
+          <p class="configuration-link" title="Click here to find your configuration file"><a href="/docs/configuration/" target="_blank"><i class="icon-question" />Find your configuration</a>
```
Comment (balloob) [inline plugins/configuration.rb]:
```suggestion
          <p class="configuration-link" title="Click here to find your configuration file"><a href="/docs/configuration/" target="_blank"><i class="icon-question" /> find your configuration</a>
```

Comment (ludeeus) [reply plugins/configuration.rb]: Why lowercase it?

Comment (balloob) [reply plugins/configuration.rb]: I felt it looked weird, but still playing with it. Want to explore adding it to the top.

Comment (ludeeus) [reply plugins/configuration.rb]: I added it to the top first, did not look good IMO

**Review by frenck — APPROVED** [review]

**Pull request merged**